app/views/core_induction_programmes/years/_module_list.html.erb

```diff
@@ -11,7 +11,7 @@
         </div>
 
         <div class="app-task-list__section">
-          <span><%= course_module.course_lessons.count.humanize.capitalize %> self-study sessions</span>
+          <span><%= course_module.course_lessons.count.humanize.capitalize %> self-study session<%= course_module.course_lessons.count === 1 ? "" : "s" %></span>
```

Comment:
We are not in js land, `===` is not what you want 😂 
`==` is equality, `===` is type conformance, I think.

Reply:
Also, use `pluralize`, 
```
<%= "session".pluralize(course_module.course_lessons.count) %>
```